package.json

```diff
@@ -382,7 +382,8 @@
     "prettier": "2.8.8",
     "puppeteer": "19.6.0",
     "ws": "8.17.1",
-    "**/@mdx-js/react/@types/react": "^17"
+    "**/@mdx-js/react/@types/react": "^17",
+    "**/prismjs": "^1.30.0"
```

Comment:
react-syntax-highlighter retractor dependency pins prismj js to 1.17 which is problematic. dont wanna touch v0 and v8 packages which would force new unwanted release.